rally-domain/src/main/groovy/yakworks/rally/mail/mailgun/MailgunService.groovy

```diff
@@ -66,7 +66,7 @@ class MailgunService extends EmailService {
             log.debug("Mail gun response : domain:$domain, id:${resp.id}, message:${resp.message}")
             return Result.OK().payload([id: resp.id, message: resp.message])
         } catch(FeignException e){
-            log.error("Mailgun Send Failure", e)
+            log.error("Mailgun failed to send email to ${mailMsg.to}", e)
```

Comment:
log more so we can debug it, goal is to diagnose the 500's right?, and we should log what we set to `Message message` right, thats whats being sent over. the way this is not wont help us sort anything out that I can see.